persistence/gov.go

```diff
@@ -2,6 +2,7 @@ package persistence
 
 import (
 	"encoding/hex"
+
```

Comment:
Why the newline here?

Reply:
Go imports are automatically formatted by my vscode Golang extension based on the "latest best practices".

For me it's an auto-edit when I do a cmd+save.

I looked online (https://stackoverflow.com/a/38714480/768439) and seems like the best practice that the linter implements is grouping by package (e.g. github vs std) separated by a new line.

<img width="755" alt="Screen Shot 2022-08-17 at 11 38 54 AM" src="https://user-images.githubusercontent.com/1892194/185182611-8d3cc95a-1c5b-4a07-96fe-bc94945be83c.png">